src/main/java/org/generationcp/middleware/domain/dms/Enumeration.java

```diff
@@ -65,6 +65,14 @@ public String getDescription() {
 		return this.description;
 	}
 
+	public int getRank(){
```

Comment:
Is this change needed still?

Reply:
Hi @darla-leafnode, it's no longer needed. I'll just revert the commit.